### PR TITLE
Gives the PDA flashlight an action button

### PR DESCRIPTION
**Pull request by Tacolizard**
:cl: Tacolizard
add: Your PDA flashlight now has an action button to toggle it.
/:cl:

### CODE REVIEW

File: code/game/objects/items/devices/PDA/PDA.dm [hunk 1]
```diff
@@ -107,6 +108,22 @@ GLOBAL_LIST_EMPTY(PDAs)
 		return attack_self(M)
 	return
 
+/obj/item/device/pda/ui_action_click()
```
Comment (RemieRichards) [inline code/game/objects/items/devices/PDA/PDA.dm]: No.
```ui_action_click()``` is banned, implement things properly on ```/toggle_light```

Comment (Tacolizard) [inline code/game/objects/items/devices/PDA/PDA.dm]: I can't find a single other item that doesn't use ui_action_click() as an example for how to do that

Comment (RemieRichards) [inline code/game/objects/items/devices/PDA/PDA.dm]: check normal actions then, it's no different.
```ui_action_click()``` is a shitty hold-over from before action datums, and it needs to die.

**Comment by improvedname** [comment]
POWERCREEP

**Comment by kevinz000** [comment]
bloat

**Comment by Tacolizard** [comment]
#goonhasitwhydontwe

**Comment by Cyberboss** [comment]
Yeah, having everyone have an action essentially all the time seems too much :-1:

**Comment by thefastfoodguy** [comment]
ya know, we need an action button UI overhaul.

**Comment by lordpidey** [comment]
Honestly, at this point, it might be worth it to somehow merge all activatable flashlights into a single UI button.  It will cut down on clutter.

**Comment by YakumoChen** [comment]
Or just use them in your hand, it's not difficult.

**Comment by Tacolizard** [comment]
@YakumoChen That isn't true, flashlights use action buttons

**Comment by Tacolizard** [comment]
To people who think this will clutter the screen, remember you can rearrange action buttons and even hide them.

**Comment by Jalleo** [comment]
Really getting it merged down IS better because of less clicking. That is outside of this PR though.

File: code/game/objects/items/devices/PDA/PDA.dm (hunk 1 above)
Comment (Tacolizard) [inline code/game/objects/items/devices/PDA/PDA.dm]: what do you mean by normal actions?

**Comment by Tacolizard** [comment]
closing this pr because nobody likes it and I can't figure out how not to use ui_action_click

**Pull request closed without merging**